README.md

```diff
@@ -226,15 +226,7 @@ supported build environments that are most likely to work.
 
 ## Getting Help
 
-The Rust community congregates in a few places:
-
-* [Stack Overflow] - Direct questions about using the language.
-* [users.rust-lang.org] - General discussion and broader questions.
-* [/r/rust] - News and general discussion.
-
-[Stack Overflow]: https://stackoverflow.com/questions/tagged/rust
-[/r/rust]: https://reddit.com/r/rust
-[users.rust-lang.org]: https://users.rust-lang.org/
+The Rust community congregates in a few places, you can find them at the [community tab of the Rust website](https://www.rust-lang.org/community)
```

Comment:
```suggestion
The Rust community congregates in a few places, you can find them
at the [community tab of the Rust website](https://www.rust-lang.org/community).
```

Reply:
Don't see much of a point in this

Reply:
You forgot to add `.`, the line length is a bonus point.